app/views/tasks/shared/_task_notes.html.erb

```diff
@@ -13,6 +13,10 @@
         <h3 class="govuk-heading-s govuk-!-padding-top-0"><%= note.user.full_name %></h3>
         <p class="govuk-body"><%= render_markdown(note.body) %></p>
 
+        <% if policy(note).edit? %>
+          <%= govuk_link_to "Change", edit_project_note_path(@task.project, note) %>
```

Comment:
Another string which could live in the language file.

Reply:
I've also made it 'Edit note'